packages/frontend/src/redux/actions/staking.js

```diff
@@ -419,47 +437,48 @@ export const handleGetLockup = (accountId) => async (dispatch, getState) => {
 
 export const handleStakingUpdateAccount = (recentlyStakedValidators = [], exAccountId) => async (dispatch, getState) => {
     const { accountId, balance } = exAccountId
-        ? getState().allAccounts[exAccountId]
-        : getState().account;
+        ? selectAllAccountsByAccountId(getState(), { accountId: exAccountId })
+        : selectAccountSlice(getState());
 
     const validatorDepositMap = await getStakingDeposits(accountId);
 
-    if (!getState().staking.allValidators.length) {
+    if (!selectStakingAllValidatorsLength(getState())) {
         await dispatch(staking.getValidators(null, exAccountId));
     }
 
-    const validators = getState().staking.allValidators
+    const validators = selectStakingAllValidators(getState())
         .filter((validator) => Object.keys(validatorDepositMap).concat(recentlyStakedValidators).includes(validator.accountId))
         .map((validator) => ({ ...validator }));
 
     await dispatch(staking.updateAccount(balance, validators, accountId, validatorDepositMap));
 };
 
 export const handleStakingUpdateLockup = (exAccountId) => async (dispatch, getState) => {
-    const { contract, lockupId: account_id } = getState().staking.lockup;
-    const { accountId } = getState().account;
+    const { contract, lockupId: account_id } = selectStakingLockup(getState());
```

Comment:
It may be out of scope for this PR but for my money it would be a little clearer for `account_id` to keep the `lockupId` name (or something like `lockupAccountId`). At a glance it's not clear what the difference between `accountId` and `account_id` is.

Reply:
Done.